### PR TITLE
Fix: #239 Update webview2-com and windows crates

### PR DESCRIPTION
**Pull request by wravery**
**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Documentation
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes. Issue #___
- [x] No

**The PR fulfills these requirements:**

- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

**Comment by wravery** [comment]
This should fix #239 (linking the issue).

File: src/platform_impl/windows/util.rs [hunk 1]
```diff
@@ -193,7 +191,7 @@ pub fn adjust_window_rect_with_styles(
     status_map(|r| {
       *r = rect;
 
-      let b_menu: BOOL = GetMenu(hwnd).is_null().into();
+      let b_menu: BOOL = (GetMenu(hwnd).0 == 0).into();
```
Comment (amrbashir) [inline src/platform_impl/windows/util.rs]: I don't know when did we change this but it should check for non-null. 
```suggestion
      let b_menu: BOOL = (GetMenu(hwnd).0 != 0).into();
```

**Review by amrbashir — CHANGES_REQUESTED** [review]

**Review by amrbashir — APPROVED** [review]
Thank you, we are grateful for your help.

**Review by wusyong — APPROVED** [review]

**Comment by lucasfernog** [comment]
Is anyone doing the same on wry and tauri? I can do it if no one is already on it.

**Comment by wravery** [comment]
Yes, I've got a branch for wry and tauri itself ready. I was just waiting for the merge commit ID so I can update the Cargo.toml references to tao.

**Pull request merged**